webview/src/experiments/components/table/body/ExperimentStatusIndicator.tsx

```diff
@@ -21,10 +21,41 @@ type ExperimentStatusIndicatorProps = {
   id: string
 }
 
+const OnRemote: React.FC<{ id: string; isStudioConnected: boolean }> = ({
+  id,
+  isStudioConnected
+}) => {
+  if (isStudioConnected) {
+    return (
+      <CellHintTooltip
+        tooltipContent={'Experiment on remote\nClick to copy Studio link'}
+      >
+        <div
+          className={styles.upload}
+          {...clickAndEnterProps(() => copyStudioLink(id))}
```

Comment:
I wish we could adjust the button design when the user copies the link but I suppose there isn't a way for the frontend to detect that it's copied 🤔

Reply:
could change it on click, but to what?

Reply:
Have you tried a checkmark icon that goes back to the link after a sec, like we do with experiment cell copy buttons?

Reply:
Updated to this:

https://github.com/iterative/vscode-dvc/assets/37993418/e11ca03e-e07d-49ba-9a63-c3844b99e992

I did try using a check instead but there are two issues there:

1. The icon becomes a checkbox momentarily
2. Unsure of the result of copying